server/routes/tokenRouter.js

```diff
@@ -0,0 +1,13 @@
+const { Router } = require('express')
+
+const {
+  postTokenController,
+  deleteTokenController,
+} = require('../Controller/tokenController')
+
+const router = Router()
+
```

Comment:
I do not think it was necessary to make a token router. It would have been better to make this a middleware to be used with every request. There is a lot of repeated code because of this. For instance, in the `userController` you decode the token both in the post and delete requests could have been written and used as a middleware for the userRouter instead.
However, given that your hands are full, we can do this later as optimization.